### PR TITLE
Merge 8.0.0-rc branch into 8.0.x

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.0.x
| Description?      | Merge 8.0.0-rc branch into 8.0.x
| Type?             | improvement
| Category?         | ME
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | ~
| Related PRs       | ~
| How to test?      | ~
| Possible impacts? | ~

### CODE REVIEW

File: composer.lock [hunk 1]
```diff
@@ -12570,5 +12570,5 @@
     "platform-overrides": {
         "php": "7.2.5"
     },
-    "plugin-api-version": "2.2.0"
+    "plugin-api-version": "2.3.0"
```
Comment (atomiix) [inline composer.lock:12573]: I think this shouldn't have been changed :/

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -732,7 +735,7 @@ GitHub contributors:
 - Sacha Froment
 - sadlyblue
 - sagaradonis
-- sallemiines
```
Comment (FabienPapet) [inline CONTRIBUTORS.md:735]: Is this a rename or is it removed

**Review by FabienPapet — APPROVED** [review]

File: CONTRIBUTORS.md (hunk 2 above)
Comment (PrestaEdit) [reply CONTRIBUTORS.md:735]: A rename, there is the long name on L351.

Comment (jolelievre) [reply CONTRIBUTORS.md:735]: Yep we have the full name now

File: composer.lock (hunk 1 above)
Comment (jolelievre) [reply composer.lock:12573]: It's a value automatically updated by composer, I think it depends on which version you have locally while running composer so ultimately it depends on who runs it

I don't know how we can prevent this except by "forcing" all contributors to use the same composer version

**Review by zuk3975 — APPROVED** [review]

**Review by PululuK — APPROVED** [review]

**Review by nicosomb — APPROVED** [review]

**Pull request merged**